tests/acceptance/test_other_commands.py

```diff
@@ -162,3 +162,9 @@ async def test_flush_all_with_delay(self, client, key_generation, memcached_addr
         # item should not be found.
         item = await client.get(key_and_value)
         assert item is None
+
+
+class TestVersion:
+    async def test_version(self, client, memcached_address_1, memcached_address_2):
+        assert await client.version(memcached_address_1)
```

Comment:
Can we assert the version value too?

Reply:
you are right, we need to check that the expected response is returned